nnstreamer/tensor_filter/tensor_filter_nntrainer.cc

```diff
@@ -166,7 +168,7 @@ int NNTrainer::loadModel() {
   gchar *content = nullptr;
   GError *file_error = nullptr;
 
-  g_assert(model_config != nullptr);
+  g_return_val_if_fail(model_config != nullptr, -1);
```

Comment:
How about removing all the asserts like one in line 83 and possibly others.
also, I think this check must be moved to line 86 and then this will become a throw.

Reply:
Yes, I'll make a V2 and upload